controller/user.go

```diff
@@ -105,15 +112,93 @@ func convertToAppUser(request *goa.RequestData, user *authservice.User) *app.Use
 	}
 }
 
-func (c *UserController) createClient(ctx *app.ShowUserContext) (*authservice.Client, error) {
-	authEndpoint := c.config.GetAuthServiceURL()
+// ListSpaces returns the list of spaces in which the user has a role
+func (c *UserController) ListSpaces(ctx *app.ListSpacesUserContext) error {
+	client, err := newAuthClient(ctx, c.config, c.httpOptions...)
+	if err != nil {
+		return jsonapi.JSONErrorResponse(ctx, err)
+	}
+	res, err := client.ListResourcesUser(goasupport.ForwardContextRequestID(ctx), authservice.ListResourcesUserPath(), "spaces")
```

Comment:
Why "spaces"?
1. Why is it plural?
2. We should use the same resource type name which used by Auth in other API and internally which is "openshift.io/resource/space" currently. If we think that this resource type name is too long then it should be changed in the Auth service. Why we use different resource type names for the same resource types in different Auth API? This is confusing.

To be clear. It makes sense to introduce `/api/user/spaces` endpoint to WIT. What I don't like is the Auth endpoint: `/api/user/resources?type="spaces"` 

cc: @sbose78 @sbryzak

Reply:
@alexeykazakov so you would prefer something like `/api/user/resources?type=openshift.io/resource/space` in auth ? (I need to see if we can avoid surrounding `openshift.io/resource/space` with double quotes)

Reply:
(I've opened https://github.com/fabric8-services/fabric8-auth/issues/623 for this change)

Reply:
Agreed. @alexeykazakov is right - my bad. 

Yeah resource type name feels a bit long actually - but nevertheless, let's stick to the exact resource type name `/api/user/resources?type=openshift.io/resource/space` should work without quotes - just like `/api/login?redirect=https://auth/api/status` . 

You might consider encoding the param value it though.